crawler/src/main/java/com/movie/recommender/crawler/Main.java

```diff
@@ -0,0 +1,12 @@
+package com.movie.recommender.crawler;
+
+public class Main {
+    public static void main(String[] args) {
+        Films films = new Films(10);
```

Comment:
do not commit test calls in the Main class
instead, you can add proper tests to the project to verify functionality